sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1186,6 +1186,80 @@ def max_deflection(self):
         else:
             return None
 
+
+    def bending_stress(self):
+        """
+        Returns bending stress of a beam cross-section at a distance `y` from the neutral axis.
+
+        Example
+        =======
+        There is a beam of length 6 m with a u.d.l of 40 KN/m over its length. The beam has a rectangular
+        cross-section(0.3 x 0.6 m) with moment of inertia. The bending stress at a point
```

Comment:
space before `(`
"moment of inertia" does not add to the description
The description does not flow well with the code that follows. I would expect something like

Create an arbitrary beam with arbitrary <parameters E and I> of length 6:

>>> L = 6
>>> b = Beam(L, E, I)

Apply resistance at the ends:

>>> up = -1
>>> b.apply_load(R1, 0, up)
>>> b.apply_load(R2, L, up)

Apply a uniform load of 40 units along the length

>>> b.apply_load(40, 0, 0, end=L)
etc...
```

Reply:
@smichr I do agree, this format looks much better.
I think for this to be done we can open an issue as the same format will have to be then adapted for every function in the module, which would be a task itself.